docker/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+BASE_DIR=`dirname $( readlink -m $( type -p $0 ))`
+
+# Ensure your SSH private key has been added to the ssh-agent as follows:
+# ssh-add /home/$USER/.ssh/id_rsa
+
+# Example Usage:
+# ./run.sh make build
+
+docker run\
```

Comment:
@apojomovsky wish: consider jumping into bash if no argument is passed to `run.sh`.

Reply:
you're right, done!